src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -408,7 +408,9 @@ public static void GetFullPath_Windows_NormalizedLongPathTooLong()
         {
             longPath.Append(Path.DirectorySeparatorChar).Append('a').Append(Path.DirectorySeparatorChar).Append('.');
         }
-        Assert.Throws<PathTooLongException>(() => Path.GetFullPath(longPath.ToString()));
+
+        // Now no longer throws unless over ~32K
```

Comment:
This was checking the normalizes to over max path. I'll make sure we have the right test.